vector-im/element-x-android
# [Rich text editor] Update design

## Type of change


## Content

Various design tweaks (see commit history and screenshot tests for details).

## Motivation and context

- https://github.com/vector-im/element-x-android/issues/1315

## Screenshots / GIFs



## Tests

- Ensure rich text editor is enabled
- Open formatting menu

## Tested devices

- OS version(s): Android 13

## Checklist
